File: URI_1020_age_in_days.js
const lines = ["400"];

let days = Number(lines[0]);

const DAYS_PER_MONTH = 30;
const DAYS_PER_YEAR = 365;

let years = Math.floor(days / DAYS_PER_YEAR);
days %= DAYS_PER_YEAR;
let months = Math.floor(days / DAYS_PER_MONTH);
days %= DAYS_PER_MONTH;

console.log(years + " ano(s)");
console.log(months + " mes(es)");
console.log(days + " dia(s)");

File: URI_1020_age_in_days.java
import java.util.Scanner;

public class URI_1020_age_in_days {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();

		final int DAYS_PER_MONTH = 30;
		final int DAYS_PER_YEAR = 365;

		int years = n / DAYS_PER_YEAR;
		n -= years * DAYS_PER_YEAR;
		int months = n / DAYS_PER_MONTH;
		n -= months * DAYS_PER_MONTH;
		int days = n;

		System.out.println(years + " ano(s)");
		System.out.println(months + " mes(es)");
		System.out.println(days + " dia(s)");

		scanner.close();
	}
}
